user/tarsnap/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "tarsnap"
+pkgver = "1.0.40"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_gen = []
+hostmakedepends = [
+    "autoconf",
+]
```

Comment:
i think you don't need this here either (i.e. without reconfing you don't need any hostdeps)

Reply:
oh true